### ISSUE TITLE
Deprecate as insecure, replace with new API version?

### ISSUE THREAD

**Issue by inspector71** [HTPCGuides/pia-portforwarding-scripts#3]
Looks like PIA now support accessing the port forward information via an API that does not require insecure passing of credential via GET parameters.

Perhaps this script should be deprecated lest someone uses it (like I did) without properly considering the security exposure possibilities?

**Comment by HTPCGuides** [comment]
Feel free to make a pull request with the updated secure version you have working, it would be great to have this working for others too :)

**Comment by inspector71** [comment]
Agreed, as the PIA script to use their new API does not seem to factor
in setting the port in Transmission (or any other software).

Alas, shell scripting is not a strength of mine but if I come up with
something, I'll share it around.

Thanks for the current version, BTW. Hope my issue lodgement did not
come across as overly critical, if at all.

**Comment by inspector71** [comment]
Added a pull request to hopefully solve this issue. Did not realise another user had already done so in 2017.

**Comment by jon-daemon** [comment]
I can see there  are some newer scripts on the [forum](https://forum.htpcguides.com/Thread-PIA-Next-Gen-Servers-and-Port-Forwarding-Automation?pid=5749#pid5749) but I cannot register to test them. Could anyone post those scripts here to check if they work?
Is there any other simple way to get the port and set it on transmission on Debian with OpenVPN for PIA?
